### PR TITLE
Added FindTrianglePerimeter

### PR DESCRIPTION
**Pull request by tehliang**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/CollatzSequence.js [hunk 1]
```diff
@@ -26,5 +26,5 @@ export function collatz (n) {
     steps.push(n)
   }
 
-  return { result: n, steps: steps }
+  return { result: n, steps }
```
Comment (appgurueu) [inline Maths/CollatzSequence.js:29]: Unrelated change, move this to a separate PR please.

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "parser": "@babel/eslint-parser"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.17.0",
+    "@babel/eslint-parser": "^7.18.9",
```
Comment (appgurueu) [inline package.json:30]: Another unrelated change

File: package-lock.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
         "node-fetch": "3.1.1"
```
Comment (appgurueu) [inline package-lock.json:18]: More unrelated changes

File: Maths/FindTrianglePerimeter.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/*
+* A polygon with three edges and three vertices is called a triangle. It is one of the fundamental geometric shapes.
+* The total length of a triangle's sides makes up the perimeter.
+* Algorithm to find the perimeter of a triangle
+*/
+function findTrianglePerimeter (x, y, z) {
+  return x + y + z
```
Comment (appgurueu) [inline Maths/FindTrianglePerimeter.js:7]: This is too trivial for my taste. If `x`, `y`, `z` were points there would be some value in this, but just adding three side lengths? That's too trivial...

**Review by appgurueu — CHANGES_REQUESTED** [review]
Not sure if this qualifies, it's pretty trivial

File: Maths/test/FindTrianglePerimeter.test.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import { findTrianglePerimeter } from '../FindTrianglePerimeter'
+
+describe('findTrianglePerimeter', () => {
+  it('should return the perimeter of a triangle', () => {
```
Comment (appgurueu) [inline Maths/test/FindTrianglePerimeter.test.js:4]: redundant `it` block; does a simple sum of three values really need 4 tests?

File: Maths/CollatzSequence.js (hunk 1 above)
Comment (tehliang) [reply Maths/CollatzSequence.js]: Sorry new to git. Not sure why have changes on the Maths/CollatzSequence.js...

File: package.json (hunk 2 above)
Comment (tehliang) [reply package.json]: Sorry new to git. Might be due to accidentally updated the @bael/eslint-parser...

File: Maths/FindTrianglePerimeter.js (hunk 4 above)
Comment (tehliang) [reply Maths/FindTrianglePerimeter.js:7]: Okay, maybe I will try to contribute to other useful algorithms that is not trivial perhaps...

File: package-lock.json (hunk 3 above)
Comment (tehliang) [reply package-lock.json:18]: Sorry new to git. Might be due to accidentely updated the @bael/eslint-parser...

File: Maths/test/FindTrianglePerimeter.test.js (hunk 5 above)
Comment (tehliang) [reply Maths/test/FindTrianglePerimeter.test.js:4]: Sorry new to git. Ya might be too many. Maybe I will find another algorithms that is not trivial....
